main.py

```diff
@@ -1 +1,17 @@
 #Mad Libs Generator Project
+verb_1 = input("Enter a verb of choice, and press enter:")
+adj_1 = input("Enter a adjective of choice, and press enter:")
+verb_2 = input("Enter second verb of choice, and press enter:")
+body_part = input("Enter a body part name and press enter:")
+adverb = input("Enter an adverb of choice, and press enter:")
+noun = input("Enter a noun of choice, and press enter:")
+verb_3 = input("Enter the third verb of choice, and press enter:")
+animal = input("Enter name of any animal of choice, and press enter:")
+noub_2 = input("Enter an noun of choice , and press enter:")
+verb_4 = input("Enter the fourth verb of choice, and press enter:")
+adj_2 = input("Enter an adjective of choice, and press enter:")
+color = input("Enter any color name, and press enter:")
```

Comment:
Consider writing a helper function or otherwise reducing the repetition of asking for user input.